src/ORM/Behavior/TreeBehavior.php

```diff
@@ -601,12 +601,12 @@ protected function _removeFromTree(EntityInterface $node)
      * Reorders the node without changing its parent.
      *
      * If the node is the first child, or is a top level node with no previous node
-     * this method will return false
+     * this method will return the same node without any changes
      *
      * @param \Cake\Datasource\EntityInterface $node The node to move
      * @param int|true $number How many places to move the node, or true to move to first position
      * @throws \Cake\Datasource\Exception\RecordNotFoundException When node was not found
-     * @return \Cake\Datasource\EntityInterface|false $node The node after being moved or false on failure
+     * @return \Cake\Datasource\EntityInterface|false $node The node after being moved or false if given number is < 1
```

Comment:
```suggestion
     * @return \Cake\Datasource\EntityInterface|false $node The node after being moved or false if `$number` is < 1
```

Reply:
I adjusted the text for both appearances in `moveUp` and `moveDown`